nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/datasetbrowser/DatasetBrowserController.java

```diff
@@ -82,6 +94,20 @@ public Stage getStage() {
     public void show() {
         stage.toFront();
         stage.show();
+        layoutStackPane();
+    }
+
+    /**
+     * Bind the height property of the datasetBrowserTabPane to the minHeightProperty of the StackPane. This will
+     * ensure the filterTextField can be centred within the tab pane header area.
+     */
+    private void layoutStackPane() {
+        if (!stackPane.minHeightProperty().isBound()) {
+            Region headerArea = (Region) datasetBrowserTabPane.lookup(".tab-header-area");
+            if (headerArea != null) {
+                stackPane.minHeightProperty().bind(headerArea.heightProperty());
+            }
+        }
```

Comment:
this is a bit sketchy but it allows the text field to be centred within the header because its parent stackpane has the same height as the the tab header (bottom picture of screenshot) vs being aligned to the top.

![image](https://github.com/nanalysis/nmrfx/assets/64663455/d123a3c9-98ec-404b-8f7f-67fc1cce01ef)

Reply:
Is it necessary to have the filter at the same level than the tabs? I was thinking it would be inside them, on the same levels as index/FID/... buttons. That way, we can change this behavior for another tab if we want too.

But if that's a request from Greg, then please keep it like this.

Reply:
The jira issue screen shot had it the same level as the tabs, and it said to display at the same level as the tab components, I will double check with him whether its necessary or if we can move it to the same level as the buttons